app/controllers/guides_controller.rb

```diff
@@ -1,8 +1,21 @@
 class GuidesController < ApplicationController
 
-  def index
-    @guide_groups = GuidesGroup.all
-    @guide_group = GuidesGroup.find_by_slug!(params.fetch(:group, 'platform'))
+  def faq
+    @article_list = FaqGroup.all
+    @article = FaqGroup.find_by_slug!(params.fetch(:group))
+    @title = "FAQs"
+    @description = "Frequently asked questions about #{@article.name.downcase} on Assembly."
+    @show_toc = false
+    render :index
+  end
+
+  def guides
+    @article_list = GuidesGroup.all
+    @article = GuidesGroup.find_by_slug!(params.fetch(:group, 'platform'))
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.